knative-operator/pkg/apis/operator/v1alpha1/knativekafka_types.go

```diff
@@ -30,6 +30,10 @@ type KnativeKafkaSpec struct {
 	// HighAvailability allows specification of HA control plane.
 	// +optional
 	HighAvailability *base.HighAvailability `json:"high-availability,omitempty"`
+
+	// Set logging configuration of the data plane.
+	// +optional
+	Logging Logging `json:"logging,omitempty"`
```

Comment:
should this be a pointer since it's optional?
```suggestion
	Logging *Logging `json:"logging,omitempty"`
```